app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command_split = command.split()
+    if len(command_split) != 3:
+        return
+    tag, source, destination = command_split
+    if len(destination.split("/")) == 1:
```

Comment:
The code uses a hardcoded forward slash as the path separator. This is common on UNIX and UNIX-like systems, but Windows traditionally uses a backslash . A more cross-platform approach would be to use os.path.join() and os.path.split()